ql/experimental/math/hybridsimulatedannealing.hpp

```diff
@@ -96,8 +96,6 @@ namespace QuantLib {
           resetScheme_(resetScheme), resetSteps_(resetSteps == 0 ? QL_MAX_INTEGER : resetSteps),
           localOptimizer_(localOptimizer),
           optimizeScheme_(localOptimizer != nullptr ? optimizeScheme : NoLocalOptimize) {
```

Comment:
I think you're changing the behavior—previously the default scheme was `NoLocalOptimize` (because the local optimizer was null), now it's `EveryBestPoint`.  Do we want to do this?

Reply:
I'm restoring the original behavior after it was accidentally changed here: https://github.com/lballabio/QuantLib/commit/ce770680be4e51f32a51a66b9502e607cfeff4a3

If we decide that we want `NoLocalOptimize` by default, we can just delete the reset code in the constructor, since in that case the optimizer is not used.